pkg/store/experiment/experiment.go

```diff
@@ -72,6 +73,51 @@ func (e *experimentStore) ListMeta(_ context.Context, kind, ns, name string) ([]
 	return archives, nil
 }
 
+func (e *experimentStore) getUID(_ context.Context, kind, ns, name string) (string, error) {
+	archives := make([]*core.ArchiveExperimentMeta, 0)
+
+	if err := e.db.Table("archive_experiments").Where(
+		"namespace = ? and name = ? and kind = ?", ns, name, kind).
+		Find(&archives).Error; err != nil {
+		return "", err
+	}
+
+	if len(archives) == 0 {
+		return "", fmt.Errorf("get UID failure")
+	}
+
+	UID := archives[0].UID
+	st := archives[0].StartTime
+
+	for _, archive := range archives {
+		if st.Before(archive.StartTime) {
+			st = archive.StartTime
+			UID = archive.UID
+		}
+	}
+	return UID, nil
```

Comment:
Not really understand this. So if there are multiple archives, we only want to get the earliest UID?

Reply:
If there are multiple archives with the same name, we need to get the one with the latest creation time. 
Because the detail interface obtains a specific archive, you can select the archive you want through the UID field. But if no UID is passed, the latest archive is selected by default.